### DISCUSSION TITLE
404 Trying to add new seats

### DISCUSSION THREAD

**Discussion by chrisShick** [community/community#84126] [Enterprise]
Hi! I am an organization owner and it looks like the link to our organization billing page to add seats is not working. Is anyone else having this problem?

**Comment by wesleybertipaglia** [comment]
Hi @chrisShick, I'm also an organization owner and the payments page is currently working.

**Comment by chrisShick** [comment]
@wesleybertipaglia I can see the payments page but I can't add new seats. The url,  here https://github.com/enterprises/{organization_name}/settings/billing, isn't working. (Obviously, the {organization_name} is our org.)

**Reply by wesleybertipaglia** [reply]
@chrisShick there is the issue, try to access changing enterprises to organizations in the url, like this:

https://github.com/organizations/{organization_name}/settings/billing
